lambdas/handlers/authoriser_handler.py

```diff
@@ -72,6 +83,35 @@ def lambda_handler(event, context):
     return auth_response
 
 
+def find_login_session(ndr_session_id):
+    logger.debug("Retrieving session for session ID ending in: " + ndr_session_id[-4:])
```

Comment:
-4 does not mean much; should this be created as a var so that we know what that array position links to?

Reply:
Thanks.  This is for extracting the last 4 chars in session ID for logging purpose.
It is now extracted as a tiny helper func `redact_id` for readability